From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Subject: util_macros.h: add missing insclusion
Date: Tue, 3 Jan 2023 14:19:37 +0200

The header is the direct user of definitions from the math.h, include it.

Link: https://lkml.kernel.org/r/20230103121937.32085-1-andriy.shevchenko@linux.intel.com
Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/util_macros.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/util_macros.h~util_macrosh-add-missing-insclusion
+++ a/include/linux/util_macros.h
@@ -2,6 +2,8 @@
 #ifndef _LINUX_HELPER_MACROS_H_
 #define _LINUX_HELPER_MACROS_H_
 
+#include <linux/math.h>
+
 #define __find_closest(x, a, as, op)					\
 ({									\
 	typeof(as) __fc_i, __fc_as = (as) - 1;				\
_
